packages/tinacms/src/admin/components/Sidebar.tsx

```diff
@@ -36,7 +36,7 @@ const Sidebar = ({ cms }: { cms: TinaCMS }) => {
   const [menuIsOpen, setMenuIsOpen] = React.useState(false)
 
   const isLocalMode = cms.api?.tina?.isLocalMode
-  const navBreakpoint = 1000
```

Comment:
Should we consider setting this breakpoint in a configuration file and then using it in the Tailwind config to ensure consistency with our existing breakpoints like "sm," "md," and "lg"? Or is this localised to this file only?

Reply:
Yeah its localised to this file only. I was thinking of moving it to css but that would require extra refactoring effort